### ISSUE TITLE
missing path templates in kms

### ISSUE THREAD

**Issue by xiaozhenliu-gg5** [googleapis/gapic-generator-typescript#205]
one path template is missing in kms API which breaks original samples-test
```
      cryptoKeyPathPathTemplate: new gaxModule.PathTemplate(
        'projects/{project}/locations/{location}/keyRings/{key_ring}/cryptoKeys/{crypto_key_path=**}'
      ),
```
it's listed in `googleapis/googleapis/google/cloud/kms/v1/cloudkms_gapic.yaml`, but not referenced anywhere in proto, while it's tested in sample test.

Reference: https://github.com/googleapis/nodejs-kms/pull/264

**Comment by xiaozhenliu-gg5** [comment]
can I get some advice here @alexander-fenster ? should we keep this path template or since it's not referenced and we can remove it, also the related sample test? 
Thanks!

**Comment by alexander-fenster** [comment]
If I'm reading this correctly, the proto annotations must be fixed in this case to add a missing resource. We can discuss it tomorrow with @noahdietz to make sure I'm getting the idea.

**Comment by noahdietz** [comment]
Yes the annotation for this resource type is missing. It seems it must be a file-level annotation because there is no defining message.

However, this resource name pattern is the same as the `cryptoKey` pattern except for the very last pattern variable name and that it includes the wildcard `**`.

```
projects/{project}/locations/{location}/keyRings/{key_ring}/cryptoKeys/{crypto_key_path=**}
projects/{project}/locations/{location}/keyRings/{key_ring}/cryptoKeys/{crypto_key}
```

While they are quite similar, they are semantically different. The `**` indicates that `/` is allowed in the value of the pattern variable.

Let's add a file-level annotation in `google/cloud/kms/resource.proto` to support this.

**Comment by xiaozhenliu-gg5** [comment]
Thanks for the context @noahdietz, I will add the file-level annotation to see if it works. And also the `getIamPolicy` messages is not defined in the protos[it's not defined anywhere], but tested in the original samples-test, do you know if we removed them intentionally?

**Comment by xiaozhenliu-gg5** [comment]
Fixed by cl/291420695
